src/graphcast_agent/waku_handling.rs

```diff
@@ -48,36 +44,31 @@ pub fn build_content_topics(
 /// Makes a filter subscription from content topics and optionally pubsub topic
 /// Strictly use the first of pubsub topics as we assume radios only listen to one network (pubsub topic) at a time
 pub fn content_filter_subscription(
```

Comment:
perhaps update this function name to be `content_filter` and utilize it whenever you are creating a content filter